js/songs/_song_show.html

```diff
@@ -0,0 +1,11 @@
+<div class="song">
+  <h2><a data-ui-sref="songShow({id: song.id})">{{song.title}}</a></h2>
+  <h4>By: {{song.artist}}</h4>
+  <div ng-show="type === 'show'">
```

Comment:
I like what you're doing here so that the detailed into only shows sometimes, but I don't like the property name `type` for this. It made since to use `type` on your form (new vs edit), but here, it's not the type of song, it's whether or not we're showing detail... I'd use a boolean value like `showDetail` which would be true or false.